website/docs/guides/index.md

```diff
@@ -1,3 +1,9 @@
+---
+sidebar_position: 1
+---
+
+# Introduction
+
 MCAP (pronounced "em-cap") is a modular container file format for heterogeneous timestamped data. It is ideal for robotics applications, as it can record multiple streams of structured and unstructured data (e.g. ROS, Protobuf, JSON Schema, etc.) in a single file.
 
 ## Benefits
```

Comment:
Agreed, I was thinking we should clean this up in a separate PR. I could take a look in this one but the PR is already so massive and it would be easier to review copy changes in smaller chunks.